packages/app-utils/src/components/ThemeBootstrap.tsx

```diff
@@ -21,11 +23,17 @@ export function ThemeBootstrap({ children }: ThemeBootstrapProps): JSX.Element {
     [pluginModules]
   );
 
+  const settings = useSelector<RootState, ReturnType<typeof getSettings>>(
```

Comment:
Use `useAppSelector` instead, it should get typed automatically: https://github.com/deephaven/web-client-ui/blob/58ee88dc92bfe9a283ebc789c93f23639a954ba3/packages/dashboard/src/redux/selectors.ts#L29

Reply:
Nice. It was annoying that `useSelector<RootState>(getSettings)` didn't infer the return type